components/team/TeamSection.vue

```diff
@@ -0,0 +1,13 @@
+<template>
+  <div
+    class="text-neutral-800 text-2xl font-semibold my-4 max-md:text-4xl"
+  >
+    {{ departmentName }}
+  </div>
+</template>
+
+<script setup>
+defineProps({
```

Comment:
you should add a `withDefaults` around here and set a very generic default `departmentName` like "Other team" to make sure this is safe